aem-nextjs-app/pages/product-details/index.jsx

```diff
@@ -0,0 +1,68 @@
+import Head from 'next/head';
```

Comment:
can we use the new `app` folder routing from next13 instead? lets not start creating pages in `pages` folder (next12 way)